frontend/src/metabase/query_builder/components/DatasetEditor/EditorTabs/EditorTabs.tsx

```diff
@@ -1,57 +1,55 @@
-import type { IconName } from "metabase/ui";
+import { t } from "ttag";
+
 import { Icon } from "metabase/ui";
 
-import { TabBar, Tab, RadioInput } from "./EditorTabs.styled";
+import { RadioInput, Tab, TabBar } from "./EditorTabs.styled";
 
 type Props = {
   currentTab: string;
-  options: {
-    id: string;
-    name: string;
-    icon: IconName;
-    disabled?: boolean;
-  }[];
+  disabledMetadata: boolean;
   onChange: (optionId: string) => void;
 };
 
-function EditorTabs({ currentTab, options, onChange, ...props }: Props) {
-  const inputId = "editor-tabs";
-
+export function EditorTabs({ currentTab, disabledMetadata, onChange }: Props) {
   return (
-    <TabBar {...props}>
-      {options.map(option => {
-        const selected = currentTab === option.id;
-        const id = `${inputId}-${option.id}`;
-        const labelId = `${id}-label`;
-        return (
-          <li key={option.id}>
-            <Tab
-              id={labelId}
-              htmlFor={id}
-              selected={selected}
-              disabled={option.disabled}
-            >
-              <Icon name={option.icon} />
-              <RadioInput
-                id={id}
-                name={inputId}
-                value={option.id}
-                checked={selected}
-                onChange={() => {
-                  onChange(option.id);
-                }}
-                aria-labelledby={labelId}
-                disabled={option.disabled}
-                data-testid={id}
-              />
-              <span data-testid={`${id}-name`}>{option.name}</span>
-            </Tab>
-          </li>
-        );
-      })}
+    <TabBar>
+      <li>
+        <Tab htmlFor="editor-tabs-query" selected={currentTab === "query"}>
```

Comment:
I see that `Tab` is a `label`. Do we even need `htmlFor` and `id`?

Reply:
I thought so too, but apparently sometimes it's not enough:

> Note: Do not to nest `<input>` inside of `<label>` because some assistive technology (such as [Dragon NaturallySpeaking](https://en.wikipedia.org/wiki/Dragon_NaturallySpeaking)) do not support it.


[source](https://www.a11yproject.com/posts/how-to-write-accessible-forms/)